File: app/src/main/java/com/example/securityangel/utils/SecurityConstants.kt
package com.example.securityangel.utils

object SecurityConstants {
    const val RISK_PASSWORD_LEAK = "risk_password_leak" // סיסמה דלפה (מהוולט)
    const val RISK_MALICIOUS_APP = "risk_malicious_app" // אפליקציה מסוכנת
    const val RISK_UNSECURE_WIFI = "risk_unsecure_wifi" // ווי-פיי ציבורי
    const val RISK_OUTDATED_OS = "risk_outdated_os"     // מערכת הפעלה ישנה

    const val STATUS_SAFE = "safe"
    const val STATUS_AT_RISK = "at_risk"
}

File: SecurityAngelIOS/SecurityAngelIOS/Logic/SecurityConstants.swift
import Foundation

/// Mirrors Android's `SecurityConstants.kt`. Used as keys in the user
/// document's `activeRisks` field.
enum SecurityRisk: String {
    case passwordLeak = "risk_password_leak"
    case maliciousApp = "risk_malicious_app"
    case unsecureWifi = "risk_unsecure_wifi"
    case outdatedOS   = "risk_outdated_os"
    case jailbroken   = "risk_jailbroken_device"
}

enum SecurityStatus: String {
    case safe   = "safe"
    case atRisk = "at_risk"
}
